Core/src/Vertexing/AdaptiveMultiVertexFinder.cpp

```diff
@@ -91,7 +91,11 @@ Acts::Result<std::vector<Acts::Vertex>> AdaptiveMultiVertexFinder::find(
           "Could not prepare for fit. Discarding the vertex candindate.");
       allVertices.pop_back();
       allVerticesPtr.pop_back();
-      break;
+      if (m_cfg.doNotBreakWhileSeeding) {
+        continue;
+      } else {
+        break;
+      }
```

Comment:
These lines are uncovered according to sonar. If it's easy to add, it'd be nice to cover them. Otherwise we override.

Reply:
I don't see a great way of testing this right now but maybe while continue working on this.

In general this is not great and I would like to just have `continue` here. This is one of the places where we bake into the AMVF that the seed finder works iteratively and depends on track removal. Potentially a seeder can have its own state and not depend on track removal similar to the truth one here by just providing the second highest peak in the track density. But I fear that is too big of a change and will certainly effect Athena